### PR TITLE
Clarify the paragraph about slurmdbd priorities

### PR DESCRIPTION
**Pull request by lproven**
Disambiguates the paragraph about converting the master slurmdbd before any others 

fixes https://github.com/SUSE/doc-hpc/issues/16

### CODE REVIEW

File: xml/slurm.xml [hunk 1]
```diff
@@ -1061,7 +1061,8 @@ success!</screen>
         <title>Convert primary <systemitem class="daemon">slurmdbd</systemitem> first</title>
         <para>
          If a backup database daemon is used, the primary one needs to be
-         converted first. The backup will not start until this has happened.
+         upgraded and run to perform the conversion first. The backup will not
+         start until this has happened.
```
Comment (sknorr) [inline xml/slurm.xml:1065]: "the backup" is ambiguous to me, so:

```suggestion
         upgraded and run to perform the conversion first. The backup
         <systemitem class="daemon">slurmdbd</systemitem> will not
         start until this has happened.
```

File: xml/slurm.xml [hunk 2]
```diff
@@ -1061,7 +1061,8 @@ success!</screen>
         <title>Convert primary <systemitem class="daemon">slurmdbd</systemitem> first</title>
         <para>
          If a backup database daemon is used, the primary one needs to be
```
Comment (sknorr) [inline xml/slurm.xml:1063]: While we're at it, let's make this as boring, synonym-free, and graspable as possible:
```suggestion
         If a backup <systemitem class="daemon">slurmdbd</systemitem> daemon is used,
         the primary  <systemitem class="daemon">slurmdbd</systemitem> needs to be
```

**Review by sknorr — COMMENTED** [review]
Wfm, mostly. :)

File: xml/slurm.xml (hunk 1 above)
Comment (lproven) [reply xml/slurm.xml]: @e4t -- any comments?

**Pull request merged**